jujugui/static/gui/src/app/components/entity-details/content/readme/readme.js

```diff
@@ -93,18 +121,48 @@ YUI.add('entity-content-readme', function() {
     */
     _getReadmeCallback: function(error, data) {
       if (error) {
-        console.error(error); 
+        console.error(error);
         this.setState({readme: 'No readme.'});
       } else {
         var readme = data;
-        this.setState({readme: this.props.renderMarkdown(readme)});
+        const renderMarkdown = this.props.renderMarkdown;
+        let renderer = new renderMarkdown.Renderer();
+        renderer.heading = (text, level) => {
+          const escapedText = text.toLowerCase().replace(/[^\w]+/g, '-');
```

Comment:
Could we have two similar texts and therefore two identical ids?
Should we include an incremental number in the escaped text?